res/res_config_pgsql.c

```diff
@@ -1551,7 +1570,6 @@ static int parse_config(int is_reload)
 
 	ast_verb(2, "PostgreSQL RealTime reloaded.\n");
 
-	/* Done reloading. Release lock so others can now use driver. */
```

Comment:
Why this removal?

Reply:
vim :D .

I fixed the indentation with vim.

Restored.